proto/types.proto

```diff
@@ -71,4 +71,6 @@ message Envelope {
     bytes plain = 13;
     bytes cipher = 14;
   }
+
+  optional string relays = 17;
```

Comment:
if we gonna re-introduced this again why not reuse the old `federation` field. @muhamadazmy @AbdelrahmanElawady

Reply:
yes, it is currently used to set the relay of twin destination in `rmb-sdk-go` [here](https://github.com/threefoldtech/tfgrid-sdk-go/blob/a1f093abfdf10606bdf01ec8e86b88be807eb830/rmb-sdk-go/peer/peer.go#L448). so it would break compatibility with older versions.

Reply:
also for the delay part, for clients that are using this new feature there should be no delay as cache will be updated with the requests made. for clients that are not using this field and changing relays there should be a delay up to 6 seconds until new blocks are produced.

This is already an improvement to the currently used relay warmer that adds a delay up to 10 minutes and previously before warmer it could take up to 60 seconds.